inst/textdata/namelist.tsv

```diff
@@ -494,59 +494,73 @@ caves	nl	grotten	NA
+ep_08.4_explan	nl	Toename van stoffen (excl. eutrofiërende of verzurende stoffen), in zodanige mate dat dit leidt tot een achteruitgang van de levensgemeenschap. De stoffen komen de standplaats binnen via het oppervlaktewater, inclusief overstromingswater en runoff (aanvoer van stoffen, veelal met sediment - modderstromen inbegrepen).	NA
 ep_08.5	nl	8.5 Thermische verontreiniging: toename temperatuur oppervlaktewater	08.5_verontr_therm
 ep_08.5_explan	nl	Een toename van de watertemperatuur ter hoogte van de beschouwde aquatische of oeverlevensgemeenschap, in zodanige mate dat deze er negatief door wordt beïnvloed. De temperatuursverhoging is daarbij het gevolg van warm lozingswater	NA
 ep_09.1	nl	9.1 Geluid en trillingen	09.1_geluid
-ep_09.1_explan	nl	Menselijke activiteit die een gedragswijziging veroorzaakt en/of tot een verhoging van de mortaliteit leidt waardoor de natuurlijke dynamiek van populaties nadelig beïnvloed wordt.	NA
+ep_09.1_explan	nl	Een toename van geluid, infra- of ultrasone vormen van trillingen of druk in lucht, bodem en/of water die een gedragswijziging veroorzaakt en/of tot een verhoging van de mortaliteit leidt waardoor de natuurlijke dynamiek van populaties nadelig beïnvloed wordt.	NA
 ep_09.2	nl	9.2 Licht en straling	09.2_licht
+ep_09.2_explan	nl	Een wijziging van het natuurlijke stralingsniveau, door kunstmatige stralingsbronnen, die een gedragswijziging veroorzaakt en/of tot een verhoging van de mortaliteit leidt waardoor de natuurlijke dynamiek van populaties nadelig beïnvloed wordt.	NA
 ep_09.3	nl	9.3 Beweging en andere visuele verstoring	09.3_beweging
+ep_09.3_explan	nl	Een verstoring louter door de aanwezigheid en/of beweging van mensen, dan wel voorwerpen die niet thuishoren in een natuurlijke omgeving, die een gedragswijziging veroorzaakt en/of tot een verhoging van de mortaliteit leidt waardoor de natuurlijke dynamiek van populaties nadelig beïnvloed wordt.	NA
 ep_101	nl	101 Klimaatverandering in droge perioden	101_klim_droog
-ep_101_explan	nl	Een toename van temperatuur, wind en CO2 in de atmosfeer en een afname resp. toename van de neerslaghoeveelheden, inclusief extreme events. Dit treedt op gedurende een voor de levensgemeenschap betekenisvolle periode (bv. maand / seizoen), in zodanige mate dat deze er negatief door wordt beïnvloed, en waarbij de effecten supplementair zijn boven die van milieudrukken, ontstaan op gebiedsniveau. Dit met uitsluiting van rechtstreekse invloeden van zeespiegelstijging.	NA
+ep_101_explan	nl	Een toename van temperatuur, wind en CO2 in de atmosfeer en een afname van de neerslaghoeveelheden, inclusief extreme events. Dit treedt op gedurende een voor de levensgemeenschap betekenisvolle periode (bv. maand / seizoen), in zodanige mate dat deze er negatief door wordt beïnvloed, en waarbij de effecten supplementair zijn boven die van milieudrukken, ontstaan op gebiedsniveau. Dit met uitsluiting van rechtstreekse invloeden van zeespiegelstijging.	NA
```

Comment:
Het is voor mij een moeilijk vatbare milieudruk, omdat ze in belangrijke mate/hoofdzakelijk indirect werkt via de andere milieudrukken. Ik zou het hier beperken tot de directe impact : (acute) droogte, verminderde kieming van zaden met een kiemrust of een verhoogde ziektegevoeligheid door een afname van het aantal vorstdagen

Reply:
@w-jan heb dit ook even met @cecileherr teruggekoppeld. De formulering is bewust ruim gehouden, de essentie in het onderscheid met andere milieudrukken zit in: "waarijb de effecten supplementair zijn boven die van milieudrukken, ontstaan op gebiedsniveau". Dez definitie werd gebruikt voor de expertenscores voor habitattypes, dus we zouden de scope ook daarom niet veranderen of beperken.